packages/forms/src/elements/select/select.tsx

```diff
@@ -1,183 +1,110 @@
-					<div>
-						<StyledInputLabel
-							element="label"
-							isError={meta && meta.touched && meta.error}
-							cfg={cfg}
-							{...getLabelProps()}
-						>
-							<InputElementHeading
-								label={label}
-								required={required}
-								hint={hint}
-								meta={meta}
-								accessibilityHelper={helper}
-							/>
-							<Flex className={styles.relative + ' ' + elementStyles.select}>
-								<Input
-									ref={ref}
-									autoComplete="off"
-									type="text"
-									testId={testId}
-									label={label}
-									disabled={disabled}
-									placeholder={placeholder}
-									readOnly={readOnly}
-									required={required}
-									onClick={() => toggleMenu()}
-									className={styles.input}
-									accessibilityHelper={helper}
-									onKeyPress={rest.onKeyPress}
-									{...getInputProps()}
-								/>
-								{showToggleButton && (
-									<button
-										type="button"
-										disabled={disabled}
-										aria-label="open-dropdown"
-										data-testid={`${testId}-button`}
-										className={styles.iconButton}
-										onClick={() => toggleMenu()}
-										{...getToggleButtonProps()}
-									>
-										<UnfoldMore />
-									</button>
-								)}
-							</Flex>
-						</StyledInputLabel>
-						<Flex className={styles.relative + ' ' + elementStyles.select}>
-							<div
-								{...getMenuProps({
-									className: classNames([
-										{ [styles['popup-isopen']]: isOpen },
-										styles.popup,
-									]),
-								})}
-							>
-								{isOpen && (
-									<PopupBox
-										searchable={!readOnly}
-										{...{
-											getItemProps,
-											inputValue,
-											options,
-											highlightedIndex,
-											selectedItem,
-											handleNotFoundButtonClick,
-											notFoundMessage,
-										}}
-									/>
-								)}
-							</div>
-						</Flex>
-					</div>
-				)}
-			</Downshift>
+			/>
 		);
 	},
 );
 
 export const FFSelect: React.FC<
 	FieldProps & Omit<SelectProps, 'children'>
 > = React.forwardRef<
-	HTMLInputElement,
+	HTMLSelectElement,
 	FieldProps & Omit<SelectProps, 'children'>
 >((fieldProps, ref) => {
 	return (
 		<Field
 			{...fieldProps}
 			required={fieldProps.required}
 			ref={ref}
-			render={({
-				input,
-				initialSelectedItem,
-				itemToString = (item: any) => (item ? item.label : ''),
-				onChange,
-				...props
-			}: any) => {
-				return (
-					<Select
-						ref={ref}
-						initialSelectedItem={initialSelectedItem}
-						itemToString={itemToString}
-						onChange={(value, _ctx) => {
-							// override onChange from outside if needed
-							if (value && onChange && typeof onChange === 'function') {
-								return onChange(value, input.onChange);
-							}
-							// otherwise forward
-							return input.onChange(value);
-						}}
-						{...props}
-					/>
-				);
+			render={({ input, initialSelectedItem, ...props }: any) => {
+				return <Select ref={ref} {...props} />;
```

Comment:
False positive, probably because it's the same name as an HTML element. `...props` includes a `label` prop which is connected to the select element by an `id` prop.